app/routes/hello.js

```diff
@@ -0,0 +1,6 @@
+export async function get(req, res) {
+    const hello = {
+      'body' : 'hello world'  
```

Comment:
`body` does not need to be quoted here. In JS representations of JSON, only the _values_ need to be quoted. There should not be a space between the key and the `:`. `eslint` should catch this, if you're using it inline in your editor, or you can run `yarn lint` in the project root.

Reply:
`yarn lint` in the project root is not reporting this error!

Reply:
Well then, boo on me. This is configured in our `.eslintrc` file. Let's make a practice of updating it to _enforce_ any style nits that come up in PRs, so that we can have an automated way of catching those and can instead spend our time talking about things of substance. If you see any rules that can/should be changed, feel free to add them: https://eslint.org/docs/rules/

Reply:
I'm confused about the action to take here, if any.

`yarn lint` isn't picking up on _any_ of your nits, in this or the other PR.

Are you saying you're going to update the lint config to catch these things?

If so, I'll just proceed.